packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -1,6 +1,6 @@
 ## The CDK Construct Library for AWS CodeDeploy
 
-### Applications
+### EC2/On-Premise Applications
```

Comment:
I like the "on-premise" spelling more. If you're going to change it, do it consistently in the entire ReadMe.

Reply:
It's a title, so i figured it was correct grammar to capitalize ...

Reply:
Sure. I don't feel strongly either way.